src/rust/madara/sources/hiperdex/res/source.json

```
@@ -0,0 +1,18 @@
+{
+  "info": {
+    "id": "en.hiperdex",
+    "lang": "en",
+    "name": "Hiperdex",
+    "version": 1,
+    "url": "https://1sthiperdex.com",
+    "nsfw": 1
```

seems pretty much nsfw
```suggestion
    "nsfw": 2
```